src/content/learn/separating-events-from-effects.md

```diff
@@ -1197,12 +1197,12 @@ export default function Timer() {
   return (
     <>
       <h1>
-        Counter: {count}
+        Sayaç: {count}
         <button onClick={() => setCount(0)}>Reset</button>
       </h1>
       <hr />
       <p>
-        Every second, increment by:
+        Saniyedeki artis miktari:
```

Comment:
Bunu digerlerini turkce karakterlerle yazarsaniz daha iyi olur.

Reply:
Hataları uyardığınız için teşekkür ederim. Yeni PR'da düzeltildi.